src/components/Item/CollectionTabs.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div v-if="!!children">
+    <v-tabs v-model="currentTab" class="mb-3" background-color="transparent">
+      <v-tab v-for="typeList in children" :key="typeList.Type">
+        {{ typeList.Type }} ({{ typeList.Items.length }})
+      </v-tab>
+    </v-tabs>
+    <v-tabs-items v-model="currentTab" class="transparent">
+      <v-tab-item v-for="typeList in children" :key="typeList.Type">
+        <v-container>
+          <skeleton-item-grid v-if="loading" :view-type="''" />
+          <item-grid :loading="loading" :items="typeList.Items" />
+        </v-container>
+      </v-tab-item>
+    </v-tabs-items>
+  </div>
+</template>
+
+<script lang="ts">
+import { BaseItemDto } from '@jellyfin/client-axios';
+import Vue from 'vue';
+import { mapGetters, mapActions } from 'vuex';
+
+export default Vue.extend({
+  props: {
+    item: {
+      type: Object,
+      required: true
+    }
+  },
+  data() {
+    return {
+      currentTab: 0,
+      loading: false,
+      breakpoints: {
+        600: {
+          visibleSlides: 2
+        },
+        960: {
+          visibleSlides: 3
+        },
+        1264: {
+          visibleSlides: 4
+        },
+        1904: {
+          visibleSlides: 5
+        }
+      }
+    };
+  },
+  async fetch() {
+    this.loading = true;
+    await this.fetchBoxsetChildren({ itemId: this.item.Id }).finally(
```

Comment:
Just FYI: Leaving this here means that, if the parent item changes (the collection item), nothing will change for this instance, as this logic is only called when the Vue instance for this component is being created (i.e when the item details page is loaded)

I don't know if the parent collection component can change at some point (by removing an item from a collection, but iirc that's not implemented yet), but I guess it's better to do things correctly from the start

You have two options:

* Move this logic to a watcher: [Vue watchers allow you to call a function when some variable changes](https://vuejs.org/v2/guide/computed.html). By default, they're executed once [the lifecycle is in mounted stage](https://es.vuejs.org/v2/guide/instance.html#Diagrama-del-Ciclo-de-vida), but you can override it and run the handler as well as soon as the component instance is created. See here: https://github.com/jellyfin/jellyfin-vue/blob/4f18f67fa9bbab78441d84fb74200207fa5db6f9/src/components/System/Snackbar.vue#L20-L28

* Keep this in the fetch hook but create a watcher for ``item`` that calls ``this.$fetch``. You can even remove ``loading`` from ``data`` and use fetchState and forget about handling the loading state yourself. [More info at nuxt docs](https://nuxtjs.org/docs/components-glossary/fetch/)